src/gui/modals/FileDialog.cpp

```diff
@@ -66,6 +89,25 @@ FileDialog::FileDialog( QWidget *parent, const QString &caption,
 		urls << QUrl::fromLocalFile( volumesDir.absolutePath() );
 #endif
 
+#ifdef LMMS_BUILD_LINUX
+
+	// FileSystem types : https://www.javatpoint.com/linux-file-system
+	QStringList usableFileSystems = {"ext", "ext2", "ext3", "ext4", "jfs", "reiserfs"};
```

Comment:
I propose to also add the following types:
* "ntfs3": For users who have mounted NTFS partitions under Linux.
* "fuse.sshfs": Access to remote file systems via SSH.

I have tested both types on my Linux machine.

The "fuse.sshfs" mount is a remote access to my smartphone via KDE Connect so adding this mount type would add quite some flexibility.

There's also another NTFS driver that's provided by [ntfs-3g](https://wiki.ubuntuusers.de/Archiv/FUSE/ntfs-3g/) but I don't know the exact name of the mount type right now. I thinks it's also something with fuse.

Reply:
I have just checked by mounting a USB stick that's formatted to NTFS using the [ntfs-3g](https://wiki.ubuntuusers.de/Archiv/FUSE/ntfs-3g/) driver. The mount type in this case is "fuseblk". So if "ntfs3" and "fuseblk" were added then most NTFS partitions would be shown.